search/includes/classes/class-search.php

```diff
@@ -11,9 +11,54 @@ class Search {
 	public const QUERY_INTEGRATION_FORCE_ENABLE_KEY = 'vip-search-enabled';
 	public const SEARCH_ALERT_SLACK_CHAT = '#vip-go-es-alerts';
 	public const SEARCH_ALERT_LEVEL = 2; // Level 2 = 'alert'
-	// Empty for now. Will flesh out once migration path discussions are underway and/or the same meta are added to the filter across many
-	// sites.
+	/**
+	 * Empty for now. Will flesh out once migration path discussions are underway and/or the same meta are added to the filter across many
+	 * sites.
+	 */
 	public const POST_META_DEFAULT_ALLOW_LIST = array();
+	/**
+	 * Jetpack Default Allow list to make migration path easier. See usage to see how is jetpack allow_list used.
+	 */
+	public const JETPACK_POST_META_DEFAULT_ALLOW_LIST = array(
```

Comment:
I chose to copy this in rather than have external dependency especially considering possible issues with different versions of JP clients can be on. What do you think?

Reply:
Thinking about this...there's definitely pros and cons to both approach, but since we're applying the `jetpack_sync_post_meta_whitelist` filter later, we already have an external dependency. Instead of hardcoding this list (which will need to be manually updated) and manually applying the filter, what if we just call `Automattic\Jetpack\Sync\Defaults::get_post_meta_whitelist()`?

The downside of this is that upstream changes in Jetpack (changes to default meta) will throw our index consistency off (if checking for full content consistency). At some point in the future, we'll (ideally) have all VIP sites using VIP Search, at which point being tied to JP in this way becomes a liability. 

Need to think this through more, I'll start a discussion thread in Slack.

Reply:
The consensus from the Slack discussion is that we should leave it as-is and not add another direct dependency on Jetpack. These change so infrequently that using JP directly would be an over-optimization, and this way there are no surprises.